src/components/SharedComponents/ButtonGroup/ButtonGroup.tsx

```diff
@@ -16,7 +16,11 @@ const ButtonGroup: FC<ButtonGroupProps> = (props) => {
    */
   const { children, ...flexProps } = props;
 
-  return <Group {...flexProps}>{children}</Group>;
+  return (
+    <Group role="group" {...flexProps}>
+      {children}
+    </Group>
```

Comment:
As per @rightsaidjames suggestion I've added `role="group"` – to do this I've hardcoded `role="group"` into the `ButtonGroup` component, which should not be problematic as it's implied by the semantics inferred from its name